server/locales/zh.json

```diff
@@ -0,0 +1,5 @@
+{
+  "auth.login.required.missing": "用户名/密码未填写",
```

Comment:
I’ve mistakenly forgotten to make nested objects.
That'll do for now, I wanted to make error messages i18n without importing ALL translation from client-side.